content/api/entity-reference/index.html

```diff
@@ -50,7 +60,7 @@
 <!--
     <div class="showcase">
         <div class="row">
-            
+
             <div class="col-sm-4">
```

Comment:
Should this be commented out? Also, I think we're using fa-gift instead of fa-money for the Gift Entity (double-check w/ @Blackbaud-HalleyCoplin)